buildSrc/src/main/java/io/micronaut/guides/feature/SwaggerAnnotations.java

```diff
@@ -0,0 +1,10 @@
+package io.micronaut.guides.feature;
+
+import jakarta.inject.Singleton;
+
+@Singleton
+public class SwaggerAnnotations extends AbstractFeature {
+    public SwaggerAnnotations() {
+        super("swagger-annotations", "swagger-annotations");
```

Comment:
I think it would be worth extracting a swagger feature from the OpenAPI feature. 

https://github.com/micronaut-projects/micronaut-starter/blob/3.3.x/starter-core/src/main/java/io/micronaut/starter/feature/other/OpenApi.java